src/routes/app/(app)/components/default-explore-page.svelte

```diff
@@ -0,0 +1,337 @@
+  import Box from '$lib/components/icons/Box.svelte';
+  import DripList from '$lib/components/icons/DripList.svelte';
+  import DripListCard, {
+    DRIP_LIST_CARD_FRAGMENT,
+  } from '$lib/components/drip-list-card/drip-list-card.svelte';
+  import filterCurrentChainData from '$lib/utils/filter-current-chain-data';
+  import isClaimed from '$lib/utils/project/is-claimed';
+  import { gql } from 'graphql-request';
+  import type {
+    DefaultExplorePageFeaturedDripListsFragment,
+    DefaultExplorePageFeaturedProjectFragment,
+  } from './__generated__/gql.generated';
+  import type { z } from 'zod';
+  import type { postsListingSchema } from '../../../api/blog/posts/schema';
+  import LatestNewsSection from './latest-news-section.svelte';
+  import ConnectWalletPrompt from './connect-wallet-prompt.svelte';
+
+  const FEATURED_PROJECT_ACCOUNT_IDS =
+    {
+      1: [
+        '80921576051643469277397866636792942368647018452892810554457309839360',
+        '80928956806149918791864723629668437820661066502202314166815319654400',
+        '80989205010981758696261160004449877944077887004065826078532843448906',
+        '80921140646830818724035150101819719966329403614944137690624336855040',
+        '80912096692731427285642748238291568975884076216206760523486136893440', // nice-node
+        '80927335273972468167722947750338907267861671542981060844246982983680', // starknet.js
+        '80927325632295926773992520689210905998818993360029926329589912567808', // snapshot
+        '80922395546375089598655709477693009806793075640399849243804470083584', // graph-node
+      ],
+      5: [
+        '81084611675088797239845552682012929720024883823846356540336220583709',
+        '80921553623925136102837120782793736893291544351678576578072673071411',
+        '81084953153801269804906196669849986124054336368266435383120426750828',
+        '80921553623925136102837120782793736893291544351678576578072673072128',
+        '80921553623925136102837120782793736893291544351678576578072673072640',
+        '80921553623925136102837120782793736893291544351678576578072673071412',
+        '80921553623925136102837120782793736893291544351678576578072673071408',
+        '80921553623925136102837120782793736893291544351678576578072673071616',
+      ],
+    }[PUBLIC_NETWORK] ?? [];
+
+  export let projects: DefaultExplorePageFeaturedProjectFragment[];
+  export let blogPosts: z.infer<typeof postsListingSchema>;
+  export let featuredDripLists: DefaultExplorePageFeaturedDripListsFragment[];
+  export let totalDrippedPrices: Awaited<ReturnType<typeof cachedTotalDrippedPrices>>;
+  export let tlv: number;
+
+  $: featuredProjects = projects.filter((p) =>
+    FEATURED_PROJECT_ACCOUNT_IDS.includes(p.account.accountId),
+  );
+
+  // 2 latest posts. Sort by date
+  $: blogPosts = blogPosts
+    .sort((a, b) => new Date(b.date).getTime() - new Date(a.date).getTime())
+    .slice(0, 2);
+
+  // Last 4 projects. TODO: sort by claim date
```

Comment:
Just noting you have a TODO here. Worth turning into a Github issue?

Reply:
ah yeah i just copied all this from the existing explore page code. Don't think this TODO actually applies anymore, will remove it